modules/installation-gcp-permissions.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+[id="installation-gcp-permissions_{context}"]
+= Required GCP permissions
+
+When you attach the `Owner` role to the service account that you create,
+you grant that service account all of the required permissions. To deploy an {product-title}
```

Comment:
all the required permissions for a specific task, or all possible permissions?

Reply:
Yes.